MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
```

Comment:
Remove images, will be added properly.
For links use like this, will add links after release  - `Crop`